js/src/lib/interfaces/Types.ts

```diff
@@ -1,19 +1,23 @@
 export const MODALITIES = [
+	"multimodal",
 	"nlp",
 	"audio",
 	"cv",
 	"rl",
+	"time_series",
 	"other",
 ] as const;
 
 export type Modality = typeof MODALITIES[number];
 
 export const MODALITY_LABELS: Record<Modality, string> = {
-	nlp:   "Natural Language Processing",
-	audio: "Audio",
-	cv:    "Computer Vision",
-	rl:    "Reinforcement Learning",
-	other: "Other",
+	multimodal:  "Multimodal",
+	nlp:         "Natural Language Processing",
+	audio:       "Audio",
+	cv:          "Computer Vision",
+	rl:          "Reinforcement Learning",
+	time_series: "Time Series",
```

Comment:
```suggestion
	time_series: "Time Series",
	structured:   "Structured Data",
```

Reply:
and actually I would advocate for time series to be inside structured

Reply:
Maybe the number of Time Series datasets/models is so small that it's maybe overkill to have a dedicated modality yet.

However in the long run I expect it to become separated to the other structured datasets we'll have. We may want to have a separate modality just for time series at one point anyway for classification, forecasting, anomaly detection, etc. As vision, audio and text modalities, time series require very specific preprocessing and model architectures.

Though I agree such datasets often come with structured metadata to help predictions

Reply:
individual tasks will still time-series related so I like using `structured` as an umbrella for a few different things, personally. Feels like a better level of generality

Reply:
What about table tasks? Technically it's also structured no? But we've kept them under NLP

Reply:
Since modality is not heavily used anywhere, I feel it's safe to change afterward if needed once we see more adoption. Let's go with `structured` and have everything together, so we don't have sections just pointing to a couple of models/datasets. Later on we can always split pragmatically.